readme.md

```diff
@@ -159,6 +159,14 @@ If present in `options` and `options.method` is not set, `options.method` will b
 
 The `content-length` header will be automatically set if `body` is a `string` / `Buffer` / `fs.createReadStream` instance / [`form-data` instance](https://github.com/form-data/form-data), and `content-length` and `transfer-encoding` are not manually set in `options.headers`.
 
+###### cookieJar
+
+Type: [`tough.CookieJar` instance](https://github.com/salesforce/tough-cookie#cookiejar)
+
+Supports cookies globally. You don't need to care of parsing nor storing them. [Click here](#cookies) for an example.
```

Comment:
`Click here` kinda links is a bad pattern. Let's make it descriptive instead. Here, it can just be `[Example.]()`.

Reply:
Yeah, true.